### ISSUE TITLE
    from bs4 import BeautifulSoup ModuleNotFoundError: No module named 'bs4'

### ISSUE THREAD

**Issue by Prat-ik** [grabowskiadrian/shopify-products-scraper#1]
from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'

**Comment by grabowskiadrian** [comment]
Did you install requirements?

You should `pip3 install -r requirements.txt` to install bs4 from BeautifulSoup package

**Comment by Prat-ik** [comment]
yes I did, still not working, I already have beautifulsoup in my package
list

**Comment by grabowskiadrian** [comment]
Please send me more information about your environment. What OS, windows/linux? Python version?
